src/app/modules/item/pages/item-edit/item-edit.component.ts

```diff
@@ -136,7 +144,13 @@ export class ItemEditComponent implements OnDestroy, PendingChangesComponent {
         enteringTimeMin: this.itemForm.get('entering_time_min'),
         enteringTimeMaxEnabled: this.itemForm.get('entering_time_max_enabled'),
         enteringTimeMax: this.itemForm.get('entering_time_max'),
-      } : {})
+      } : {}),
+      ...(this.enableTeamSection ? {
+        entryParticipantType: this.itemForm.get('entry_participant_type'),
+        entryFrozenTeams: this.itemForm.get('entry_frozen_teams'),
+        entryMaxTeamSize: this.itemForm.get('entry_max_team_size'),
+        entryMinAdmittedMembersRatio: this.itemForm.get('entry_min_admitted_members_ratio'),
+      } : {}),
```

Comment:
There are a lot of repeated transformations from snake case to lower case when using objects from the api, is there a dedicated place to declare such model mappers/transformers ?

Reply:
There is mapper for incoming data but probably this form was written before. 
here are so many things to be improved for these form (not involving many pages of the platform hopefully)... but we'll start from the non-type safety with is pretty dangerous and error prone, but the case might be fixed with the type fix.